### ISSUE TITLE
Having a leading uppercase Unicode letter in the name of a class causes an error 

### ISSUE THREAD

**Issue by vladyio** [ruby/irb#9]
**Details**
If you run one of the following examples (I'm using `irb 1.0.0 (2018-12-18)`) in console:

- `class Класс` + <kbd>Enter</kbd>
- `class Σφάλμα` + <kbd>Enter</kbd>
- `class ЖClass` + <kbd>Enter</kbd>

you'll get a SyntaxError like this:

```
SyntaxError ((irb):1: syntax error, unexpected end-of-input, expecting end)
```

So the first Unicode symbol causes this kind of error. 
At the same time code like this works fine: `class Класс; end`.

**What is expected to be**

Basically it should run like any similar piece of code:

```
2.6.0 :001 > class Kласс
2.6.0 :002?>   # ... 
2.6.0 :003?> end
```

**Comment by stomar** [comment]
It should be noted that the first example presumably does not start with ASCII „K“ but with 'CYRILLIC CAPITAL LETTER KA' (U+041A).

**Comment by vladyio** [comment]
> It should be noted that the first example presumably does not start with ASCII „K“ but with 'CYRILLIC CAPITAL LETTER KA' (U+041A).

Yes, it is a Cyrillic letter, sorry for the confusion.

**Comment by aycabta** [comment]
This is already fixed. Please try `gem install irb --pre` with Ruby 2.6.3 or 2.7.0-dev.
